.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -317,60 +189,27 @@ jobs:
-        - name: Attempt to download build files artifact
-          id: download_artifact_1
-          uses: actions/download-artifact@v3
-          continue-on-error: true
-          with:
-            name: build_files
-            path: /home/runner/work/oppia/
-        - name: Unzip build files
-          if: steps.download_artifact_1.outcome != 'failure'
+        - name: Install Python deps
           run: |
-            ls -la
-            unzip build_files.zip
-            rm build_files.zip
-            echo "Files in ./:"
-            ls -la .
-            echo "Files in oppia_tools:"
-            ls -la oppia_tools
-            echo "Files in oppia:"
-            ls -la oppia
-            echo "Files in build:"
-            ls -la oppia/build
-            echo "Files in third_party:"
-            ls -la oppia/third_party
-          working-directory: /home/runner/work/oppia
-          shell: bash
-        - uses: ./.github/actions/install-oppia-dependencies
-        - name: (Artifact re-build) Cache third_party/static
-          if: steps.download_artifact_1.outcome == 'failure'
-          uses: actions/cache@v3
-          env:
-            cache-name: cache-node-modules
-          with:
-            path: /home/runner/work/oppia/oppia/third_party/static
-            key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('dependencies.json') }}
-            restore-keys: |
-              ${{ runner.os }}-build-${{ env.cache-name }}-
-              ${{ runner.os }}-build-
-              ${{ runner.os }}-
-        - name: (Artifact re-build) Build Webpack
-          if: steps.download_artifact_1.outcome == 'failure'
-          run: python -m scripts.build --prod_env
+            pip install --require-hashes --no-deps -r requirements.txt
+            pip install --require-hashes --no-deps -r requirements_dev.txt
+        - name: Install chrome
+          run: python -m scripts.install_chrome_for_ci
+        - name: Install ffmpeg
+          run: sudo apt install ffmpeg
+          # Deleting ubuntu files for getting more space (refer: https://github.com/actions/runner-images/issues/2840#issuecomment-1284059930)
+        - name: Deleting ubuntu files
+          run: |
+            sudo rm -rf /usr/share/dotnet
+            sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+        - name: Start dev-server
+          run: |
+            sudo make build
+            sudo make run-devserver
         - name: Run Acceptance Test ${{ matrix.suite }}
-          run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_acceptance_tests --skip-build --suite=${{ matrix.suite }} --prod_env
+          run: |
+            export PATH=/home/runner/work/oppia/oppia_tools/node-16.13.0/bin:${PATH}
+            xvfb-run -a --server-args="-screen 0, 1285x1000x24" ./node_modules/.bin/jasmine --config="./core/tests/puppeteer-acceptance-tests/jasmine.json" ./core/tests/puppeteer-acceptance-tests/spec/${{ matrix.suite }}
```

Comment:
Can you run the tests using the make cmd so it' easier to read?

Reply:
done